themes/theme-ios11/components/ProductGrid/components/Item/components/ItemDetails/index.jsx

```diff
@@ -36,14 +37,23 @@ class ItemDetails extends PureComponent {
 
     return (
       <div className={styles.details}>
+        {/*
+          This feature is currently in BETA testing.
+          It should only be used for approved BETA Client Projects
+        */}
+        { isBeta() && <Swatches productId={productId} /> }
```

Comment:
MOve `isBeta` inside pls

Reply:
According to our plans discussed before `isBeta` was supposed to be used on first occurrence in the themes.

Reply:
done